.github/workflows/nfazure.yml

```diff
@@ -0,0 +1,50 @@
+name: nf-azure CI
+on:
+  workflow_dispatch:
+
+jobs:
+  build:
+    name: Build nf-nomad
+    if: "!contains(github.event.head_commit.message, '[ci skip]')"
+    runs-on: nfazure
+    timeout-minutes: 10
+    strategy:
+      fail-fast: false
+      matrix:
+        java_version: [19]
+
+    steps:
+      - name: Environment
+        run: env | sort
+
+      - name: Checkout
+        uses: actions/checkout@v1
+        with:
+          fetch-depth: 1
+          submodules: true
+
+      - name: Setup Java ${{ matrix.java_version }}
+        uses: actions/setup-java@v1
+        with:
+          java-version: ${{matrix.java_version}}
+          architecture: x64
+
+      - name: Compile
+        run: ./gradlew assemble
+
+      - name: Tests
+        run: ./gradlew check
+        env:
+          GRADLE_OPTS: '-Dorg.gradle.daemon=false'
+
+      - name: Install
+        run: ./gradlew clean unzipPlugin -x test -P version=$(git rev-parse --short "$GITHUB_SHA")
```

Comment:
@jagedn , is the overall idea to build the plug-in and test with local asset?


Maybe we can use `nfazure` for testing a proper release, what do you think?

Reply:
right now, with current PR, the idea is to run the GH action against some branch/commit but yes, can be a good idea to use `nfazure` as version name. I will change it